### PR TITLE
Fix tooltip for hotkey

### PR DESCRIPTION
**Pull request by koppor**
Before

grafik

After:

grafik

- [x] Change in CHANGELOG.md described
- [ ] Tests created for changes
- [x] Screenshots added (for bigger UI changes)
- [x] Manually tested changed features in running JabRef
- [x] Check documentation status (Issue created for outdated help page at [help.jabref.org](https://github.com/JabRef/help.jabref.org/issues)?)
- [ ] If you changed the localization: Did you run `gradle localizationUpdate`?

### CODE REVIEW

File: src/main/java/org/jabref/gui/util/KeyStrokeToString.java [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+
+import javax.swing.KeyStroke;
+
+/**
+ * Class code adapted from http://www.java2s.com/Tutorial/Java/0260__Swing-Event/ConvertingaKeyStroketoaString.htm
+ */
+public class KeyStrokeToString {
+
+    public static String keyStrokeToString(KeyStroke key) {
+        StringJoiner sj = new StringJoiner(" ");
```
Comment (lenhard) [inline src/main/java/org/jabref/gui/util/KeyStrokeToString.java]: I hate to nitpick, but could you assign more meaningful names to this and the following variable?

Comment (koppor) [reply src/main/java/org/jabref/gui/util/KeyStrokeToString.java]: I googled for that and just added it. I think, it was always broken.

Comment (Siedlerchr) [reply src/main/java/org/jabref/gui/util/KeyStrokeToString.java]: This KeyStrokeToSTring class looks really odd. I am not sure if we really need it for javafx

File: src/main/java/org/jabref/gui/entryeditor/EntryEditor.java [hunk 2]
```diff
@@ -917,7 +918,8 @@ public void actionPerformed(ActionEvent e) {
         private AutoLinkAction() {
             putValue(Action.SMALL_ICON, IconTheme.JabRefIcon.AUTO_FILE_LINK.getIcon());
             putValue(Action.SHORT_DESCRIPTION,
-                    Localization.lang("Automatically set file links for this entry") + " (Alt-F)");
+                    Localization.lang("Automatically set file links for this entry") +
+                            " (" + KeyStrokeToString.keyStrokeToString(Globals.getKeyPrefs().getKey(KeyBinding.AUTOMATICALLY_LINK_FILES)) + ")");
```
Comment (Siedlerchr) [inline src/main/java/org/jabref/gui/entryeditor/EntryEditor.java]: Why don't you call Globals.getKeyPrefs().get(KeyBinding.AUTOMATICALLY_LINK_FILES).get()=
The get Method returns already the binding name (or the default binding, e.g. ctrl+z as an Optional<String>  so no need to rely on the KeyStroke stuff

Comment (koppor) [reply src/main/java/org/jabref/gui/entryeditor/EntryEditor.java]: Did not find the method.

**Comment by koppor** [comment]
@Siedlerchr Thank you for the hint. I hope, I got the Optional thingy right

    Globals.getKeyPrefs().get(KeyBinding.AUTOMATICALLY_LINK_FILES).map(b -> " (" + b + ")").orElse(""))

**Pull request merged**